snapcraft/plugins/catkin.py

```diff
@@ -889,9 +889,11 @@ def find(self, package_name):
 
     def _run(self, arguments):
         with tempfile.NamedTemporaryFile(mode='w+') as f:
-            lines = ['export PYTHONPATH={}'.format(os.path.join(
-                self._catkin_install_path, 'usr', 'lib', 'python2.7',
-                'dist-packages'))]
+            lines = ['export PYTHONPATH={}:{}'.format(
+                os.path.join(self._catkin_install_path, 'usr', 'lib',
+                             'python2.7', 'dist-packages'),
+                os.path.join(self._catkin_install_path, 'opt', 'ros',
+                             'kinetic', 'lib', 'python2.7', 'dist-packages'))]
```

Comment:
@mrjogo I'm sorry, but I'm having trouble understanding what you're doing and what's wrong. Is this https://forum.snapcraft.io/t/python-pkg-resources-only-works-in-certain-paths/978 ? If not, perhaps it would be best to start a forum conversation about this, with some example code?